python/hsfs/feature_group.py

```diff
@@ -1049,7 +1105,7 @@ def insert(
         operation: Optional[str] = "upsert",
         storage: Optional[str] = None,
         write_options: Optional[Dict[Any, Any]] = {},
-        validation_options: Optional[Dict[Any, Any]] = {},
+        validation_options: Optional[Dict[Any, Any]] = {"save_report": True},
```

Comment:
This is potentially dangerous, cause the dict is defined when the function is defined the first time. So that means if the dict gets modified in a call to the function, it will stay modified for all future calls to the method.
This is a python thing, and we had problems with it in the past.

So please double check, if we modify it anywhere in the function call.

Reply:
I can change it to `Optional[Dict[Any, Any]] = None` and then:
```python3
if validation_options is None:
  validation_options = {'save_report': True}
```

Reply:
yes, let's do it, just to be sure